frontend/pages/hosts/details/cards/Software/HostSoftware.tsx

```diff
@@ -222,6 +248,31 @@ const HostSoftware = ({
     [id, renderFlash, refetchSoftware]
   );
 
+  const toggleSoftwareFiltersModal = useCallback(() => {
+    setShowSoftwareFiltersModal(!showSoftwareFiltersModal);
+  }, [setShowSoftwareFiltersModal, showSoftwareFiltersModal]);
+
+  const onApplyVulnFilters = (vulnFilters: ISoftwareVulnFiltersParams) => {
+    const newQueryParams = {
```

Comment:
Do we need any checks here to make sure we don't bounce to the first page if filters don't actually change, or do we not have enough info here to dirty-check that?

Reply:
No, but I can build it :)